nexus/db-queries/src/db/datastore/ip_pool.rs

```diff
@@ -541,13 +514,6 @@ mod test {
             .expect("Failed to get silo's default IP pool");
         assert_eq!(ip_pool.name().as_str(), "default-for-silo");
 
-        // if we ask for the fleet default by name, we can still get that one
-        let ip_pool = datastore
-            .ip_pools_fetch(&opctx, &Name("default".parse().unwrap()))
-            .await
-            .expect("Failed to get fleet default IP pool");
-        assert_eq!(ip_pool.id(), fleet_default_pool.id());
-
```

Comment:
can't do this anymore because the method is gone, but this integration test covers it.

https://github.com/oxidecomputer/omicron/blob/01e730acf3efeb6ff2181d558cf373ded383a365/nexus/tests/integration_tests/instances.rs#L3442-L3451